File: src/main/java/com/converter/api/dto/UserView.java
package com.converter.api.dto;

import com.converter.api.model.User;
import lombok.Getter;
import org.springframework.hateoas.RepresentationModel;

@Getter
public class UserView extends RepresentationModel<UserView> {

    private final String email;

    public UserView(User user) {
        this.email = user.getEmail();
    }
}

File: src/main/kotlin/com/converter/api/dto/UserView.kt
package com.converter.api.dto

import org.springframework.hateoas.RepresentationModel
import com.converter.api.model.User

class UserView(user: User) : RepresentationModel<UserView>() {

    val email: String

    init { // TODO: Segue a mesma linha da classe ConversionView.kt
        email = user.email
    }
}